src/Protocol/ActivityPub/Receiver.php

```diff
@@ -1313,6 +1313,11 @@ private static function getReceivers(array $activity, string $actor, array $tags
 	 */
 	private static function getReceiverForActor(array $tags, array $receivers, int $target_type, array $profile): array
 	{
+		if (empty($profile['uri-id'])) {
+			Logger::warning('Missing profile uri-id');
```

Comment:
If this is meant to get fixed automatically when the post update go through then this function should be a pass-through until it does.

Reply:
Here is the line from Friendica's application log. Do you need more?

```
E_WARNING: Undefined array key "uri-id" {"code":2,"message":"Undefined array key \"uri-id\"","file":"/var/www/html/libranet/src/Protocol/ActivityPub/Receiver.php","line":1319} - {"file":"Receiver.php","line":1319,"function":"getReceiverForActor","request-id":"656c24d07f10e","stack":"ErrorHandler::handleError (1319), Receiver::getReceiverForActor (1217), Receiver::getReceivers (371), Receiver::prepareObjectData (672), Receiver::processActivity (1650), Processor::fetchMissingActivity (819), Receiver::routeActivities (239), Queue::process (39), ProcessQueue::execute, Worker::execFunction (386), Worker::execute (121), Worker::processQueue (87)","uid":"405990","process_id":209777}
```

Reply:
At first I thought, that it was some post update thingy. But since the error is a missing index field, it can only originate from an actor that hadn't been found.

Reply:
Thanks @AlfredSK . The warning originates from line 1174. The actor wasn't found, the array is then passed the function anyway. In line 1216 we have to change the `if` statement to not check for an empty `$actor` but an empty `$profile`.